style.css

```diff
@@ -0,0 +1,187 @@
+@import url('https://fonts.googleapis.com/css?family=Poppins');
```

Comment:
In this document, I would vertically separate style declaration blocks "{}" with an empty line.